cmd/cork/create.go

```diff
@@ -115,7 +115,7 @@ func init() {
 	creationFlags.StringVar(&sdkUrlPath,
 		"sdk-url-path", "sdk", "SDK URL path")
 	creationFlags.StringVar(&sdkURL,
-		"sdk-url", "mirror.release.flatcar-linux.net", "SDK URL (supported schema: gs, https)")
+		"sdk-url", "mirror.release.flatcar.net", "SDK URL (supported schema: gs, https)")
```

Comment:
```suggestion
		"sdk-url", "mirror.release.flatcar-linux.net", "SDK URL (supported schema: gs, https)")
```

Reply:
(To prevent this you could first replace the large bulk of `github.com/flatcar-linux` and then look for any remaining `flatcar-linux` references)